stdlib/src/binascii.rs

```diff
@@ -182,6 +182,77 @@ mod decl {
         Ok((*c - 0x20) & 0x3f)
     }
 
+    #[pyfunction]
+    fn rlecode_hqx(s: ArgAsciiBuffer) -> PyResult<Vec<u8>> {
+        let runchar = 0x90; //RUNCHAR = b"\x90"
+        s.with_ref(|buffer| {
+            let len = buffer.len();
+            let mut out_data = Vec::<u8>::with_capacity((len * 2) + 2);
+
+            let mut idx = 0;
+            while idx < len {
+                let ch = buffer[idx];
+
+                if ch == runchar {
+                    out_data.push(runchar);
+                    out_data.push(0);
+                    return Ok(out_data);
+                } else {
+                    let mut inend = idx + 1;
+                    while inend < len && buffer[inend] == ch && inend < idx + 255 {
+                        inend += 1;
+                    }
+                    if inend - idx > 3 {
+                        out_data.push(ch);
+                        out_data.push(runchar);
+                        out_data.push(((inend - idx) % 256) as u8);
+                        idx = inend - 1;
+                    } else {
+                        out_data.push(ch);
+                    }
+                }
+                idx += 1;
+            }
+            Ok(out_data)
+        })
+    }
+
+    #[pyfunction]
+    fn rledecode_hqx(s: ArgAsciiBuffer) -> PyResult<Vec<u8>> {
+        let runchar = 0x90; //RUNCHAR = b"\x90"
+        s.with_ref(|buffer| {
+            let len = buffer.len();
+            let mut out_data = Vec::<u8>::with_capacity(len);
+            let mut idx = 0;
+
+            if buffer[idx] == runchar {
+                out_data.push(runchar);
+            } else {
+                out_data.push(buffer[idx]);
+            }
```

Comment:
```suggestion
            out_data.push(if buffer[idx] == runchar {
                runchar
            } else {
                buffer[idx]
            });
```
`if` is an expression in rust

Reply:
isn't this same as below?
```suggestion
            out_data.push(buffer[idx]);
```

Reply:
> `if` is an expression in rust

I was never seen this! thanks :)